test/dsets.c

```diff
@@ -3915,7 +3915,7 @@ test_nbit_compound_2(hid_t file)
         unsigned int v;
         char         b[2][2];
         atomic       d[2][2];
-    } complex;
```

Comment:
This structure had to be renamed since "complex" is a keyword from complex.h and conflicted